server/service/src/login.rs

```diff
@@ -154,6 +155,14 @@ impl LoginService {
                 });
             }
         };
+
+        // Check that the logged in user has access to at least one store on the site
+        match user_service.find_user(&user_account.id) {
```

Comment:
i wish this was named better (find_user_active_on_this_site)

Reply:
Hm yeah, I thought that too - I think I will rename, its a bit confusing 🙏